site/src/pages/TasksPage/TasksPage.tsx

```diff
@@ -44,7 +44,7 @@ const TasksPage: FC = () => {
 		refetchInterval: 10_000,
 	});
 	const idleTasks = tasksQuery.data?.filter(
-		(task) => task.current_state?.state === "idle",
+		(task) => task.current_state?.state === "idle" && task.status === "active",
```

Comment:
An alternative solution is handling this on the backend instead and unsetting current state. Thoughts?

This is probably good enough for now though.

Edit: On second thought, I'll give the BE approach a try.

Reply:
That said, I think the task state would need a different type or some change to better reflect the state. I don't think `idle` makes sense for a stopped task, even if that's the last app state the workspace had.

For now, though, the FE approach will suffice.